systemservices/marketplace/src/tasks/preparePublishServiceVersion.ts

```diff
@@ -1,19 +1,39 @@
 import { TaskInputs, TaskOutputs } from "mesg-js/lib/service"
 import { Marketplace } from "../contracts/Marketplace"
-import { stringToHex, CreateTransaction, hashToHex } from "../contracts/utils";
+import { stringToHex, CreateTransaction } from "../contracts/utils";
 import { Manifest } from "../types/manifest";
+import { getService, isServiceExist } from "../contracts/service";
+import * as assert from "assert";
 
 export default (
   marketplace: Marketplace,
   createTransaction: CreateTransaction
 ) => async (inputs: TaskInputs, outputs: TaskOutputs): Promise<void> => {
   try {
+    // check inputs
+    const sid = inputs.service.definition.sid
+    assert.ok(sid.length >= 1 && sid.length <= 63, 'sid must have a length between 1 and 63') // See Core service validation (https://github.com/mesg-foundation/core)
```

Comment:
```suggestion
    assert.ok(sid.length >= 1 && sid.length <= 63, 'sid's length must be 1 at min and 63 at max') // See Core service validation (https://github.com/mesg-foundation/core)
```

between word is kind of confusing here because grammatically it says that sid's length cannot be 1 nor 63 it should only in between. and this is not correct